sktime/markov/sample.py

```diff
@@ -212,7 +213,7 @@ def by_state(dtrajs, nsample, subset=None, replace=True):
     ----------
     dtrajs : List[np.ndarray]
         underlying discrete trajectories
-    nsample : int
+    n_samples : int
```

Comment:
could be smaller than what? (see line below)

Reply:
could be smaller than n_samples

Reply:
ohhhhhh. my bad. no need to change.